### DISCUSSION TITLE
The examples have CSS classes applied, but where is the CSS?

### DISCUSSION THREAD

**Discussion by rabidkitten** [jscottsmith/react-scroll-parallax#268] [General]
I'm using Next.js and the examples don't work without me copying the CSS from some Storybook page and applying it... I've looked at the instructions for installing it and I don't see anything about copying CSS. Can someone help me?

**Comment by jscottsmith** [comment]
You don't need any CSS for installation, but some examples require CSS for the appropriate layout.

I can help if you provide more details, like which example you are trying to implement. And a CodeSandbox would be the best way to get some assistance.

**Comment by jscottsmith** [comment]
You don't need any CSS for installation, but some examples require CSS for the appropriate layout.

I can help if you provide more details, like which example you are trying to implement. And a CodeSandbox would be the best way to get some assistance.

**Comment by jscottsmith** [comment]
You don't need any CSS for installation, but some examples require CSS for the appropriate layout.

I can help if you provide more details, like which example you are trying to implement. And a CodeSandbox would be the best way to get some assistance.

**Comment by rabidkitten** [comment]
I ended up translating the classes in the example to styles. For example, the insert: 0, position: absolute instead of the CSS class names provided in the example. This way I didn't have to import all that css. Thanks.
